Source/Scene/Camera.js

```diff
@@ -3181,6 +3181,23 @@ Camera.prototype.cancelFlight = function () {
   }
 };
 
+/**
+ * Complete the current camera flight if one is in progress
+ * The camera immediately move to destination
+ */
+Camera.prototype.completeFlight = function () {
+  if (defined(this._currentFlight)) {
+    this.cancelFlight();
```

Comment:
There's still an issue here where calling `cancelFlight` will cause the current flights `cancelled` callback to be called.  Instead of calling cancelFlight, just add `this._currentFlight = undefined;` at the end of this `if` block.

Reply:
Sorry, I was wrong, what I meant was call `this._currentFlight.cancelTween();` directly here.